api/src/main/java/org/openmrs/validator/PersonValidator.java

```diff
@@ -163,5 +167,21 @@ private void rejectDateIfBefore120YearsAgo(Errors errors, Date date, String date
 			errors.rejectValue(dateField, "error.date.nonsensical");
 		}
 	}
+
+	/**
+	 * Rejects a death date if it is before birth date
+	 * 
+	 * @param errors the error object
+	 * @param deathdate the date to check
+	 * @param birthdate to check with
+	 * @param dateField the name of the field
+	 */
+	private void rejectDeathDateIfBeforeBirthDate(Errors errors, Date deathdate, Date birthdate, String dateField) {
+		if (deathdate.before(birthdate)) {
+			errors.rejectValue(dateField, "error.deathdate.before.birthdate");
```

Comment:
Did you remember to include this new error message code in the messages.properties file?